README.md

```diff
@@ -1,4 +1,9 @@
-﻿# Supported Windows Server, version 1709 amd64 tags
+﻿# Supported Windows Server, version 1803 amd64 tags
```

Comment:
We've been dropping the term 'Supported' across our .NET Docker readmes as it is rather meaningless.  Dropping it makes it a little easier to quickly scan and differentiate the sections.

Reply:
oops, I merged too quick before I see this comment.

I will create a new PR.